### ISSUE TITLE
Create a control that sets / unsets the map's `static` attribute

### ISSUE THREAD

**Issue by prushforth** [Maps4HTML/Web-Map-Custom-Element#1051]
A control that reflects the map's `static` attribute would be useful for page authors who want to include one or more maps on an HTML page and to avoid the "slipping on a banana" effect of scrolling the mouse wheel while the cursor rests inadvertently over one of those maps.

The control could work similarly to how a layer's `checked` attribute is reflected and managed by the layer control checkbox for that layer's entry. 

Should there be an event emitted when the user toggles the static control?  

The control should be opt-in i.e. not present by default but rendered when the user includes `controlslist=static`, I think.

Ideas / opinions below please
